src/face_detect.py

```diff
@@ -1,16 +1,40 @@
 import numpy as np
 from cv2 import cv2
 
-def detect_function():
+def image_detect():
```

Comment:
Try making the function prototype
def detect(image): to take in an argument and make it more modular